src/modules/thirdParty/sagas.ts

```diff
@@ -236,8 +241,8 @@ export function* thirdPartySaga(builder: BuilderAPI, catalyst: CatalystClient) {
     }
   }
 
-  function* handleUpdateApprovalFlowProgress(action: { progress: number }) {
-    yield put(updateApprovalFlowProgress(action.progress))
+  function* handleUpdateApprovalFlowProgress(action: { progress: number; tpAction: ThirdPartyAction }) {
+    yield put(updateThirdPartyActionProgress(action.progress, action.tpAction))
```

Comment:
I think, as we're doing sagas integration tests, that we should be able to place a `put` in the tests to be sure that the update is being done, would you mind doing that?

Reply:
Good point! Added!